src/HashBus.Projector.MostMentioned/MostMentionedProjection.cs

```diff
@@ -20,17 +21,17 @@ public MostMentionedProjection(IRepository<string, IEnumerable<Mention>> mention
             this.mentions = mentions;
         }
 
-        public void Handle(TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             if (!message.Tweet.UserMentions.Any())
             {
-                return;
+                return Task.FromResult(0);
             }
 
             var trackMentions = this.mentions.GetAsync(message.Tweet.Track).GetAwaiter().GetResult().ToList();

```

Comment:
make the Handle method `async` and remove `.GetAwaiter().GetResult()`, making it:

```
( await this.mentions.GetAsync(message.Tweet.Track) ).ToList();
```